src/datasource.js

```diff
@@ -38,9 +41,12 @@ export default class OCIDatasource {
 
     this.compartmentsCache = [];
     this.regionsCache = [];
+    this.tenancyconfigCache = [];
+
+    // this.getTenancyConfig();
 
-    this.getRegions();
-    this.getCompartments();
+    // this.getRegions();
```

Comment:
any reason why these are commented out??

Reply:
this part of the code is still WIP